lib/msal-common/src/uCache/serialize/Serializer.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccessTokenEntity } from "../entities/AccessTokenEntity";
+import { IdTokenEntity } from "../entities/IdTokenEntity";
+import { RefreshTokenEntity } from "../entities/RefreshTokenEntity";
+import { AccountEntity } from "../entities/AccountEntity";
+import { AppMetadataEntity} from "../entities/AppMetadataEntity";
+import { CacheHelper } from "../utils/CacheHelper";
+import {
+    AccessTokenCacheMaps,
+    IdTokenCacheMaps,
+    RefreshTokenCacheMaps,
+    AccountCacheMaps,
+    AppMetadataCacheMaps
+} from "./JsonKeys";
+
+export class Serializer {
```

Comment:
This seems like an overly generic class name for what the class is doing, is there a more specific name?

Reply:
This class no longer exists